blueprint/security/system-security-plan.md

```diff
@@ -430,28 +430,28 @@ This section does not include specific subsections as the information is the sam
 
 ### Applicability to CloudSystem
 
-Not applicable to the CloudSystem as this is an Agency’s responsibility. An Agency’s implementation of personnel security controls should cover the CloudSystem system.
+Technical controls relating to personnel security are applicable to the CloudSystem. An Agency’s implementation of personnel security controls should include the CloudSystem.
 
 ### CloudSystem compliance approach
 
-The CloudSystem provides a role-based access control implementation and associated operations guide to enable an Agency to easily and securely control access, including privileged and emergency access, to Azure and Office 365 services.
+The CloudSystem provides a role-based access control implementation and associated operations guide to enable an Agency to easily and securely control access, including privileged and emergency access, to Azure and Office 365 services. The CloudSystem leverages in-built authentication logging provided by the platform, and centralises logs to prevent unauthorised modification and deletion.
 
 ### Security controls provided by the CloudSystem
 
-* The CloudSystem provides a framework for identity and access management for Azure and Office 365 resources.
-* The CloudSystem includes automation to disable inactive accounts after 45 days.
+* All unprivileged access attempts are logged in Azure AD Sign-ins. Azure AD logs are forwarded to a Log Analytics workspace for long-term secure retention.
```

Comment:
Advanced Audit (Permium) would cover this now rather than LAW retention? https://docs.microsoft.com/en-us/microsoft-365/compliance/auditing-solutions-overview?view=o365-worldwide

Reply:
Will investigate the use of Advanced Audit/Purview as part of an upcoming update.